tests/assets/__init__.py

```diff
@@ -1,9 +1,17 @@
 from pathlib import Path
+from typing import List
 
 
 def get_asset(name: str) -> Path:
     try:
         path = next(Path('.').rglob(name))
     except StopIteration:
-        raise FileNotFoundError
+        raise FileNotFoundError(name)
     return path
+
+
+def find_asset(name: str) -> List[Path]:
+    paths = list(Path('.').rglob(name))
+    if not paths:
+        raise FileNotFoundError(name)
+    return paths
```

Comment:
about the `.meca` files, they need to be in Git LFS:
https://github.com/libero/jats-ingester/blob/master/.gitattributes

Reply:
amend the original commit because removing a file and putting it in LFS makes `git diff` and other commands go crazy

Reply:
updated.